modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/config/DefaultBalloonEditorConfiguration.js

```diff
@@ -43,6 +43,27 @@ const DEFAULT_BALLOON_EDITOR_CONFIG = {
 			element: 'code',
 			name: Liferay.Language.get('computer-code'),
 		},
+		{
+			attributes: {
+				class: 'overflow-auto portlet-msg-info',
+			},
+			element: 'div',
+			name: Liferay.Language.get('info-message'),
+		},
+		{
+			attributes: {
+				class: 'overflow-auto portlet-msg-alert',
+			},
+			element: 'div',
+			name: Liferay.Language.get('alert-message'),
+		},
+		{
+			attributes: {
+				class: 'overflow-auto portlet-msg-error',
+			},
+			element: 'div',
+			name: Liferay.Language.get('error-message'),
+		},
```

Comment:
@diegonvs This code is fine, but we want to add these styles to the default BE configuration. Looking at [requirement docs](https://docs.google.com/document/d/1QSSzjd9dDxJ5Wug-379Klx1B6rgH7-6VNybw2lD24ro/edit#heading=h.ze2euljig91a), defaults should be h1, h2, pre, cite and code.

We should add the three message styles through Blogs-specific configuration.

Reply:
> We should add the three message styles through Blogs-specific configuration.

So, it will be implemented during contentEditor migration :)

Reply:
Yes! Closing.